odoo/addons/base/models/res_company.py

```diff
@@ -182,6 +183,39 @@ def _onchange_parent_id(self):
                 if self[fname] != self.parent_id[fname]:
                     self[fname] = self.parent_id[fname]
 
+    @api.depends('country_id')
+    def _compute_uninstalled_l10n_module_ids(self):
```

Comment:
this compute function only calculates the auto-install ready modules without recursion.
It is already good enough for the business logics since the `button_install` will handel the recursion.
but it would be better to comment somewhere to clarify the result without reading the SQL